src/cargo/util/toml.rs

```diff
@@ -680,6 +681,12 @@ fn process_dependencies<F>(cx: &mut Context,
             }
             TomlDependency::Detailed(ref details) => details.clone(),
         };
+
+        if details.version.is_none() && details.path.is_none() && details.git.is_none() {
+            warnings.push(format!("warning: Dependency ({}) specified without providing a local \
```

Comment:
Ah yeah this looks ok to me, we typically avoid capital letters and ending punctuation, however